README.md

```diff
@@ -222,10 +228,10 @@ puppet preview --last --view overview --nodes diff_nodes
 ####Process output
 
 All output (except reports intended for human use) is written in JSON format to allow further processing with tools like 'jq' (JSON query). The output is written to a subdirectory named after the node of the directory appointed
-by the setting `preview_outputdir` (defaults to `$vardir/preview`):
+by the option `--preview-outputdir` (defaults to `$vardir/preview`):
```

Comment:
not sure which section - MAIN I guess, but preview may run in master mode, in which case it is MASTER. (I may work on both MAIN and MASTER). (The text is on line 457 (in PR)).